### PR TITLE
Docs: Moved series doclets

### PR DESCRIPTION
**Pull request by bre1470**
This branch contains the doclet movements.

---
- To see the specific tasks where the Asana app for GitHub is being used, see below:
  - https://app.asana.com/0/0/1206591292618185
  - https://app.asana.com/0/0/1206591292618181

### CODE REVIEW

File: ts/Series/BoxPlot/BoxPlotSeriesOptions.d.ts [hunk 1]
```diff
@@ -14,31 +14,376 @@
...
+     * @sample {highcharts} highcharts/plotoptions/box-plot-styling/
+     *         Box plot styling
+     * @sample {highcharts} highcharts/css/boxplot/
+     *         Box plot in styled mode
+     *
+     * @type {Highcharts.DashStyleValue}
+     * @default Solid
+     * @since 8.1.0
+     * @product highcharts
+     * @apioption plotOptions.boxplot.boxDashStyle
```
Comment (TorsteinHonsi) [inline ts/Series/BoxPlot/BoxPlotSeriesOptions.d.ts:70]: Can we now safely remove the `@apioption` tag? 🤔

Comment (bre1470) [reply ts/Series/BoxPlot/BoxPlotSeriesOptions.d.ts:70]: I have to discuss this with Markus. We might need these in the meantime until we can retire the JSDoc plugin.

Comment (bre1470) [reply ts/Series/BoxPlot/BoxPlotSeriesOptions.d.ts:70]: I will remove the apioption tags and Markus will reconstruct these for the JSDoc plugin. The type naming might need some hard-coded mapping, but in general it is detailed enough to build the option path. Only problem are camel case names in interfaces like `SeriesBoxPlotDataLabelOptions` etc.

**Comment by TorsteinHonsi** [comment]
@bre1470 Is this for real ready to me merged into the master? Is the tooling updated so that when we run `gulp jsdoc` for the next release, it will pick up the doclets at their new locations?

**Comment by bre1470** [comment]
@TorsteinHonsi Sorry for the confusion. This was a mistake. It is not ready at all.

**Comment by TorsteinHonsi** [comment]
Okay, no worries!

**Review by TorsteinHonsi — APPROVED** [review]
Thanks!

**Pull request merged**